hazelcast/src/main/java/com/hazelcast/internal/management/events/EventMetadata.java

```diff
@@ -45,7 +45,10 @@ public EventMetadata(EventType type, long timestamp) {
         WAN_CONFIGURATION_ADDED(8),
         ADD_WAN_CONFIGURATION_IGNORED(9),
         WAN_SYNC_IGNORED(10),
-        WAN_CONFIGURATION_EXTENDED(11);
+        WAN_CONFIGURATION_EXTENDED(11),
+        RELOAD_STARTED(12),
```

Comment:
maybe prefix the new constants with `CONFIGURATION_` so it's clearer what reload refers to?

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/5476f75165b6415f444350306daa8803ebfe4902